beams/service/rpc_client.py

```diff
@@ -167,18 +167,25 @@ def run(self, command: str, **kwargs) -> HeartBeatReply:
             response = self.get_heartbeat()
             return response
 
+        # TODO: deal with none as tree name
+        tree_name = kwargs.get("tree_name") or ""
+        tree_uuid = kwargs.get("tree_uuid") or ""
```

Comment:
Nitpick: maybe using "" in the arg here is more clear than using an or
```suggestion
        tree_name = kwargs.get("tree_name", "")
        tree_uuid = kwargs.get("tree_uuid", "")
```